src/components/Messages/Message/index.js

```diff
@@ -113,14 +113,22 @@ const resolveWebRTCEndCallMessage = ({ webRtcCallEndTs, ts }) => {
 	return `${ I18n.t('Call ended at %{time}', { time }) } ${ I18n.t(' - Lasted %{callDuration}', { callDuration }) }`;
 };
 
+const getClosingMessaageText = (conversationFinishedText) => {
+	const { postChatUrl, config: { messages: { escalatedConversationFinishedText } } } = store.state;
+	if (postChatUrl) {
+		return escalatedConversationFinishedText;
+	}
+	return conversationFinishedText || I18n.t('Chat finished');
+};
+
```

Comment:
@bhardwajaditya @ear-dev This will only show one message at a time.

i.e 
For escalated: `Thank you for chatting with us. How did we do today?`
For non escalated: `Chat ended`

@ear-dev Are we okay with that for now?

Reply:
@Shailesh351 cleared this with Steve yesterday. He said we want only one of these two

Reply:
Ohh okay. Thanks @bhardwajaditya

Reply:
@bhardwajaditya is correct.......  Steve would prefer just one of the messages depending on escalated vs. non-escalated.